### DISCUSSION TITLE
Editing shortcuts

### DISCUSSION THREAD

**Discussion by qqrm** [FreeTubeApp/FreeTube#3182] [Q&A]
Everything is fine, but there are not enough options for hotkeys. Unfortunately, they only work in one layout, which is inconvenient. And what is even more inconvenient - hard-coded hotkeys. Maybe there is an option to assign other settings to hotkeys? Browser extensions allow it, video players too. It's frustrating to have to use the right hand wheel to rewind and change speed. There is still a little lack of a button to reset the speed to default (specified in the settings).
In general, I would like the functionality provided by the Video Speed Controller extension. It's nicely made.
![image](https://user-images.githubusercontent.com/4681325/218288331-067b7878-bfec-484f-9970-6498c95eef86.png)

P.S. Things are pretty basic, it's strange that I didn't find relevant discussions.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @qqrm, there is actually an open issue for this https://github.com/FreeTubeApp/FreeTube/issues/251
